haechan45th/src/services/userService.js

```diff
@@ -25,7 +35,23 @@ const signUp = async (name, email, password, profileImage) => {
     }
 };
 
+const signIn = async (email, password) => {
+    try {
+        const user = await userDao.getUserByEmail(email);
+        if (!user || !await bcrypt.compare(password, user.password)) {
+            throw new Error('KEY_ERROR');
+        }
+        const token = jwt.sign({ email: user.email }, config.JWT_SECRET_KEY, { expiresIn: "1h" });
+        return {
+            token,
+            user
+        };
```

Comment:
```suggestion
        return token
```
controller에서 말씀드렸던 것 처럼 user data는 보내지 않도록 해주세요~